File: 01.JAVA/pg-226-230/pg_226_05.java
import java.util.Scanner;

public class pg_226_05
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num1, num2;
        int i = 1;
        int j = 1;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
        System.out.println("READ TWO INTEGERS AND DISPLAY ALL NUMBERS ENDED IN 4 COMPRENDS BETWEEN THEM");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJJVJVJVJV");
        System.out.println("");

        System.out.print("Write the first integer: ");
        num1 = enter.nextInt();

        System.out.print("Write the second integer: ");
        num2 = enter.nextInt();

        if(num1 < 0)
        {
            num1 *= -1;
        }

        if(num2 < 0)
        {
            num2 *= -1;
        }


        while(i <= num1)
        {
            if(i % 10 == 4)
                {
                System.out.print(i + ", ");
                }
        i++;

        }

        System.out.println("");
        
        while(j <= num2)
        {
            if(j % 10 == 4)
            {
                System.out.print(j + ", ");
            }
        j++;
            
        }
    }
}

File: 06.PHP/pg-226-230/pg_226_05.php
<?php

echo("");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPH<br>");
echo("READ TWO INTEGERS AND DISPLAY ALL THE NUMBERS ENDED IN 4 COMPRENDED BETWEEN THEM<br>");
echo("PHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPHPH<br><br>");
echo("");

$i = 1;
$j = 1;
$num1 = 25;
$num2 = 44;

echo("Write the first number: " . $num1 . "<br>");
echo("Write the second number: " . $num2 . "<br><br>");

/*
while($i <= $num1)
    {
        if($i % 10 == 4)
            {
                echo($i . ", ");
            }
        $i++;
    }

    echo("<br><br>");

    while($j <= $num2)
    {
        if($j % 10 == 4)
            {
                echo($j . ", ");
            }
        $j++;
    }
*/

for($i; $i<=$num1; $i++)
    {
        if($i % 10 == 4)
            {
                echo($i . ", ");
            }
    }

    echo("<br><br>");

    for($j; $j<=$num2; $j++)
    {
        if($j % 10 == 4)
            {
                echo($j . ", ");
            }
    }

?>